src/Shared/HttpSys/RequestProcessing/RequestHeaders.Generated.cs

```diff
@@ -73,24 +73,13 @@ public StringValues Accept
                     _flag0 |= 0x1u;
                 }
 
-                if (_Accept.Count > 0)
-                {
-                    return _Accept;
-                }
-                return StringValues.Empty;
+                return _Accept.Count > 0 ? _Accept : StringValues.Empty;
```

Comment:
This change comes for re-running the generator.
I didn't revert these lines, as the next one who runs the generateor has the same problem 😉.

The actual code change is at the end of this diff (well, actually in the tt-file)